#include <catch2/internal/catch_startup_exception_registry.hpp>